controllers/clusteroperator.go

```diff
@@ -5,11 +5,11 @@ import (
 	"fmt"
 	"os"
 
-	"k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
 	osconfigv1 "github.com/openshift/api/config/v1"
 	"github.com/openshift/library-go/pkg/config/clusteroperator/v1helpers"
+	"github.com/pkg/errors"
```

Comment:
For included packages, can we stick to the following format? I believe we're already doing this in all other files.

stdlibs

outside packages

openshift packages

Reply:
Good idea, I'll update